anago

```diff
@@ -200,6 +200,46 @@ check_acls () {
 }
 
 ###############################################################################
+# Ensures all registries that will be used during both mock and --nomock
+# runs allow write access so we don't fall over later
+# @param registries - A space separated list of registries
+#
+ensure_registry_acls () {
```

Comment:
After @david-mcmahon updated the permissions, the above step works for me as the production user.